src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php

```diff
@@ -132,6 +132,16 @@
         header('Content-Encoding: gzip');
         echo str_repeat('-', 1000);
         exit;
+
+    case '/transfer-timeout':
```

Comment:
Don't wanna use existing `/timeout-long` because of https://bugs.php.net/bug.php?id=76342